src/Sensors/CacheEventSensor.php

```diff
@@ -40,6 +40,10 @@ final class CacheEventSensor
         ForgettingKey::class,
     ];
 
+    private const IGNORED_KEYS = [
+        'illuminate:queue:restart',
+    ];
+
```

Comment:
The queue worker triggers a cache event for the `illuminate:queue:restart` key on every loop, which causes the data to be ingested on every loop, where the only record is this cache event. I decided to ignore this key to avoid ingesting useless data.

Reply:
I'm not sure about this now.

If you use the database as the cache driver, the queries still show up. Wonder if we can cook up a better way to exclude this check on each loop.